.travis.yml

```diff
@@ -21,5 +21,7 @@ after_success:
   - julia -e 'Pkg.add("Plots")'
   - julia -e 'Pkg.add("PyCall")'
   - julia -e 'Pkg.build("PyCall"); using PyCall'
-  - julia -e 'Pkg.add("PyPlot"); using Plots; pyplot()'
+  - julia -e 'Pkg.add("PyPlot")'
```

Comment:
Seems like building PyPlot should happen here rather than after Plots tries to load it.

Reply:
Ah, never mind. I see this is now outdated.